camp/camp-server/src/main/java/org/apache/brooklyn/camp/server/rest/CampServer.java

```diff
@@ -118,7 +120,12 @@ public synchronized void stop() {
     
     public Integer getPort() {
         if (server==null) return null;
-        return server.getConnectors()[0].getLocalPort();
+        try {
+            NetworkConnector networkConnector = (NetworkConnector) server.getConnectors()[0];
+            return networkConnector.getLocalPort();
+        } catch (ClassCastException ex) {
+            return null;
```

Comment:
Don't get this, why explicitly catch this exception? Lots of other places where the same cast is done, so no point really.